subworkflows/align_short/main.nf

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { BWAMEM2_INDEX } from './modules/nf-core/modules/bwamem2/index/main'
+include { ALIGN_SHORT   } from './subworkflows/local/align_short'
+
+workflow {
+
+    fasta = "${projectDir}/assets/GCA_927399515.1.subset.fasta"
+    ch_fasta  = Channel.fromPath(fasta)
+    ch_genome = ch_fasta.map { file -> [ [ id: file.baseName.replaceFirst(/.unmasked/, "").replaceFirst(/.subset/, "") ], file ] }
```

Comment:
I went back to the way it was. The two channels are here because it is an example script, shows how to create them and how different modules/subworkflows may have different requirements. 

I didn't want to define the sampleID and datatype with the `fasta` and `reads` because it just felt like too much. And either way I can't define the `read_group` outside the workflow.